### ISSUE TITLE
POA_MONITOR_PATH env variable

### ISSUE THREAD

**Issue by phahulin** [poanetwork/poa-network-monitor#2]
It looks like `POA_MONITOR_PATH` env variable is used only to tell cron the absolute path to the monitor folder, but cron can first `cd` into the required folder and then call all scripts locally, e.g.
```
*/10 * * * *  cd /home/user/poa-network-monitor; timeout -s 2 8m scripts/test-runner.sh missing-rounds-sokol
```
Can we exclude this variable from scripts?

**Comment by Natalya11444** [comment]
This variable was also used in bash scripts for getting absolute path for running from any folder, but I've found it can get location of itself, so I updated scripts in the [PR]( https://github.com/poanetwork/poa-network-monitor/pull/5)

**Comment by phahulin** [comment]
Fixed in #5
